code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -250,15 +250,44 @@
 	reagents.add_reagent(BLOOD, 20, data)
 
 /obj/item/weapon/reagent_containers/glass/bottle/random
-	name = "Random culture bottle"
-	desc = "A small bottle. Contains a random disease."
+	name = "unknown culture bottle"
+	desc = "A small bottle. Contains an unknown disease."
 	icon = 'icons/obj/chemical.dmi'
-	//icon_state = "bottle3"
+
```

Comment:
```suggestion
icon_state = "bottle_alt"
```
Just a suggestion here. This one has the same shape as bottle3, but with the dynamic fillings overlays.